modules/local/check_samplesheet.nf

```diff
@@ -0,0 +1,30 @@
+process CHECK_SAMPLESHEET {
```

Comment:
so for now this is just moved but in the future you'll make other changes?

Reply:
yes, I was in the middle of fixing some samplesheet things but needed to fix this mapping thing first